scripts/release/packager/PACKAGE.md

```diff
@@ -7,12 +7,16 @@ zipped archive of XCFrameworks.
 
 - [`jq`](https://stedolan.github.io/jq/)
 - [`xcodegen`](https://github.com/yonaskolb/XcodeGen)
-- valid `.netrc` token for sdk registry
+- Valid `.netrc` token for SDK registry
 
-Descriptions of the files in this directory is as follows:
+## Scripts
 
-### `package-mapbox-maps-xcframework.sh`
+### `package-mapbox-maps.sh`
 
+- Usage:
+  ```
+  sh package-mapbox-maps.sh
```

Comment:
I recommend `./package-mapbox-maps.sh` and letting the hashbang determine which shell to use.

Reply:
sounds good, updated.